burn-book/src/import/onnx-model.md

```diff
@@ -1,137 +1,210 @@
-of a deep learning model. This feature makes it exponentially easier to move models between
-different frameworks, thereby significantly aiding interoperability. ONNX is supported by a number
-of frameworks including but not limited to TensorFlow, PyTorch, Caffe2, and Microsoft Cognitive
-Toolkit.
+1. **Time-saving**: Avoid the need to train models from scratch, which can be time-consuming and
+   resource-intensive.
+2. **Access to state-of-the-art architectures**: Utilize cutting-edge models developed by
+   researchers and industry leaders.
+3. **Transfer learning**: Fine-tune imported models for your specific tasks, benefiting from
+   knowledge transfer.
+4. **Consistency across frameworks**: Ensure consistent performance when moving from one framework
+   to another.
 
-### Advantages of ONNX
+## Understanding ONNX
 
-ONNX stands out for encapsulating two key elements:
+ONNX (Open Neural Network Exchange) is an open format designed to represent machine learning models.
+Key features include:
 
-1. **Model Information**: It captures the architecture, detailing the layers, their connections, and
-   configurations.
-2. **Weights**: ONNX also contains the trained model's weights.
+- **Framework agnostic**: ONNX provides a common format that works across various deep learning
+  frameworks.
+- **Comprehensive representation**: It captures both the model architecture and trained weights.
+- **Wide support**: Many popular frameworks like PyTorch, TensorFlow, and scikit-learn support ONNX
+  export.
 
-This dual encapsulation not only simplifies the porting of models between frameworks but also allows
-seamless deployment across different environments without compatibility concerns.
+By using ONNX, you can easily move models between different frameworks and deployment environments.
 
-## Burn's ONNX Support: Importing Made Easy
+## Burn's ONNX Support
 
-Understanding the important role that ONNX plays in the contemporary deep learning landscape, Burn
-simplifies the process of importing ONNX models via an intuitive API designed to mesh well with
-Burn's ecosystem.
+Burn takes a unique approach to ONNX import, offering several advantages:
 
-Burn's solution is to translate ONNX files into Rust source code as well as Burn-compatible weights.
-This transformation is carried out through the burn-import crate's code generator during build time,
-providing advantages for both executing and further training ONNX models.
+1. **Native Rust code generation**: ONNX models are translated into Rust source code, allowing for
+   deep integration with Burn's ecosystem.
+2. **Compile-time optimization**: The generated Rust code can be optimized by the Rust compiler,
+   potentially improving performance.
+3. **No runtime dependency**: Unlike some solutions that require an ONNX runtime, Burn's approach
+   eliminates this dependency.
+4. **Trainability**: Imported models can be further trained or fine-tuned using Burn's native
+   training loop.
```

Comment:
> using Burn's native training loop.

Though it was already there before, I find the formulation a bit weird, especially due to that last part. You can write you own training loop if you want 🤷‍♂️ 

Maybe we can just say: "Imported models can be further trained or fine-tuned with Burn."